jaworski_cellseg/load_data_widget.py

```diff
@@ -14,53 +14,21 @@ def jwslab_load_bio_data(napari_viewer, file_path):
             channel_axis=0,
             name=[f"Channel {i}" for i in range(bio_data.shape[0])],
         )
-        return bio_data
+        return bio_data, image.metadata
     except Exception as e:
         print(e)
 
 
-def create_load_data_widget(viewer: napari.Viewer):
+def create_load_data_widget(viewer: napari.Viewer, physical_sizes: dict):
     @magicgui(
-        call_button="Select BioImage",
+        call_button="Open BioImage",
```

Comment:
As a convention maybe name the button as `Open BioImage ...` just to show that there is a dialog after clicking this button?

Reply:
Done